learning_journal/templates/update_post_template.jinja2

```diff
@@ -115,10 +114,10 @@
     <!-- /.container -->
 
     <!-- jQuery -->
-    <script src="js/jquery.js"></script>
+    <script src="../../static/js/jquery.js"></script>
 
     <!-- Bootstrap Core JavaScript -->
-    <script src="js/bootstrap.min.js"></script>
+    <script src="../../static/js/bootstrap.min.js"></script>
```

Comment:
Consider adding this into your base template instead of importing it into every template.